### ISSUE TITLE
[css22] CSS 2.2 vs CSS 2 section overstates changes

### ISSUE THREAD

**Issue by gsnedders** [w3c/csswg-drafts#818]
[This](https://drafts.csswg.org/css2/about.html#css2.2-v-css2) section implies that 2.2 has changed a lot more than it has, compared with 2.1, and makes it sound like 2.1 is an uninteroperable mess with features nobody implements, etc.

While the section more-or-less is just slightly edited from 2.1, it doesn't make any where near so much sense when the baseline REC to compare against is 2.1.

**Comment by svgeesus** [comment]
Agreed, this is the description of changes from 2.0 to 2.1. It needs to be discarded and an actual 2.1 to 2.2 changes section added. In other words, changes since the previous Recommendation.
